test_sets/External_Consent_Status_ON.txt

```diff
@@ -0,0 +1,33 @@
+./test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_on_data_consent_disallowed.lua Check_that_SDL_returns_DISALLOWED_when_disallowed_by_external_consent_entities_off_exists_externalConsentStatus_ON_and_data_consent_disallowed
```

Comment:
@dboltovskyi, name of script has external_consent_entities_on, but definition is for external_consent_entities_off.
Please clarify.